services/migrations/gitea_uploader.go

```diff
@@ -923,9 +923,8 @@ func (g *GiteaLocalUploader) CreateReviews(reviews ...*base.Review) error {
 func (g *GiteaLocalUploader) Rollback() error {
 	if g.repo != nil && g.repo.ID > 0 {
 		g.gitRepo.Close()
-		if err := models.DeleteRepository(g.doer, g.repo.OwnerID, g.repo.ID); err != nil {
-			return err
-		}
+
```

Comment:
When and how the repository could be deleted?

Reply:
"running"

Reply:
"running" -> can be canceled.

"failed" -> can be deleted.